public/i18n/en.json

```diff
@@ -18,6 +18,7 @@
   "ftm-accepting": "Accepting",
   "ftm-ppe-needed": "PPE needed",
   "ftm-facility-type": "Facility type",
+  "ftm-filter-toggle": "Toggle filter options",
```

Comment:
I feel like it could just say "Additional filters" or "Filters" and that the caret makes it obvious? Just personal preference though.

Reply:
I agree the copy should err on the side of ELI5 than assume. Love the choice of "toggle" only concern is "Toggle additional filters" is a longish phrase and worried about mobile width.

What about just colloquial "Toggle more filters" (less characters than current)

Looks good on mobile:
![image](https://user-images.githubusercontent.com/62862914/82836110-c9f9a000-9e7a-11ea-81e2-d06ae59554ac.png)

Reply:
I don't have a strong opinion on it, I choose "filter options" since that was in figma and dropped the "sort" part. If I had to choose I'd probably just go with "Toggle filters" but you tell me

Reply:
"Toggle filters/Hide Filters" + caret sounds straight forward. Let's do that!